src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+
 public class SalaryInfo {
+    static final int DATE = 0;
+    static final int NAME = 1;
+    static final int HOURS = 2;
+    static final int PRICE = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate fromDate = LocalDate.of(Integer.parseInt(dateFrom.substring(6)),
+                Integer.parseInt(dateFrom.substring(3, 5)),
+                Integer.parseInt(dateFrom.substring(0, 2)));
+        LocalDate toDate = LocalDate.of(Integer.parseInt(dateTo.substring(6)),
+                Integer.parseInt(dateTo.substring(3, 5)),
+                Integer.parseInt(dateTo.substring(0, 2)));
```

Comment:
Please, create method and use it: 
private LocalDate parseDate(String date)
Because you parse the date multiple times in code

You can also use LocalDate.parse instead if LocalDate.of
private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");